vispy/ipython/widget.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2014, 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+from IPython.html.widgets import DOMWidget
+from IPython.utils.traitlets import Unicode, Int, Bool
+from vispy.app.backends._ipynb_util import create_glir_message
```

Comment:
Ok for app.backends.ipython.

humm nevermind my idea lead to code that only work with a specific backend of vispy...

will try to fix size now. seems more complicated than it appear!